StortSpelProjekt/Game/src/Gamefiles/Components/HealthComponent.h

```diff
@@ -0,0 +1,44 @@
+#ifndef HEALTH_COMPONENT_H
+#define HEALTH_COMPONENT_H
+
+#include "../../Engine/src/ECS/Components/Component.h"
+#include "../Events/Events.h"
```

Comment:
change include to forward declaration of eventname, as instructed in the tutorial for events